protocol/synthetix/subgraph/subgraph.arbitrum-sepolia.yaml

```diff
@@ -0,0 +1,100 @@
+specVersion: 0.0.4
+schema:
+  file: ./arbitrum-one/schema.graphql
+dataSources:
+  - kind: ethereum
+    network: arbitrum-sepolia
+    name: CoreProxy
+    source:
+      abi: CoreProxy
+      address: "0x45f3822fc5F486F28F3F33B1d602995a0d4A3944"
+      startBlock: 21127604
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      file: ./arbitrum-sepolia/index.ts
+      abis:
+        - name: CoreProxy
+          file: ./arbitrum-sepolia/deployments/system/CoreProxy.json
```

Comment:
arb sepolia is stil borked afaik, suggest removing it for the time being. Testing straight on the mainnet!